docs/glossary.md

```diff
@@ -0,0 +1,204 @@
+# Glossary
+
+## Anonymous branch
+
+An anonymous branch is a chain of commits that doesn't have any
+[named branches](#branch) pointing to it or to any of its descendants. Unlike
+Git, Jujutsu keeps commits on anonymous branches around until they are
+explicitly abandoned.
```

Comment:
Perhaps add something along the lines of:

Visible anonymous branches are tracked by the [view], which stores a list of [heads] of such branches.

Reply:
Done.